# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/polybar/hotfix-cstdint.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- polybar/include/utils/color.hpp.vanilla	2026-07-30 10:53:10.235242822 +0300
+++ polybar/include/utils/color.hpp	2026-07-30 10:53:30.195792901 +0300
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <cstdlib>
+#include <cstdint>
 
 #include "common.hpp"
 
--- polybar/include/utils/string.hpp.vanilla	2026-07-30 10:51:35.660705433 +0300
+++ polybar/include/utils/string.hpp	2026-07-30 10:51:51.435015050 +0300
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <sstream>
+#include <cstdint>
 
 #include "common.hpp"
 
--- polybar/include/ipc/encoder.hpp.vanilla	2026-07-30 10:50:13.638960792 +0300
+++ polybar/include/ipc/encoder.hpp	2026-07-30 10:50:34.205353684 +0300
@@ -4,6 +4,8 @@
 #include "errors.hpp"
 #include "ipc/msg.hpp"
 
+#include <cstdint>
+
 POLYBAR_NS
 
 namespace ipc {
--- polybar/include/ipc/msg.hpp.vanilla	2026-07-30 10:49:08.933476934 +0300
+++ polybar/include/ipc/msg.hpp	2026-07-30 10:49:26.691052054 +0300
@@ -3,6 +3,7 @@
 #include "common.hpp"
 
 #include <array>
+#include <cstdint>
 
 POLYBAR_NS
 

